### ISSUE TITLE
How to get the Accuracy on validation set?

### ISSUE THREAD

**Issue by xb-chang** [ZhipengLiu6/Chalearn2017_continuous_gesture#2]
Hi, 
Thanks you for your code.

I have went through all the steps but cannot find the validation accuracy.
Now I have the valid_prediction.txt in submission folder but seem no Accuracy is shown.
How can I get the accuracy results on validation set? 

Looking forward to hear from you.

**Comment by ZhipengLiu6** [comment]
You can download the groundtruth label and related code http://www.cbsr.ia.ac.cn/users/jwan/database/ConGD_labels.zip. There are introduction about how to get the accuracy results on validation set.

**Comment by xb-chang** [comment]
Got it. Thanks.
